src/main/java/prefs/Preferences.java

```diff
@@ -104,29 +142,51 @@ public void setPanelInfo(List<PanelInfo> panelInfo) {
 
     public void addBoard(String name, List<PanelInfo> panels) {
         assert name != null && panels != null;
-        global.addBoard(name, panels);
+        sessionConfig.addBoard(name, panels);
     }
 
+    /**
+     * Retrieves a map of all the boards
+     */
     public Map<String, List<PanelInfo>> getAllBoards() {
-        return global.getAllBoards();
+        return sessionConfig.getAllBoards();
     }
 
+    /**
+     * Retrieves a list of all the board names
+     */
     public List<String> getAllBoardNames() {
         return new ArrayList<>(getAllBoards().keySet());
     }
 
+    /**
+     * Removes the board specified
+     * @param name The name of the board to remove
+     */
     public void removeBoard(String name) {
-        global.removeBoard(name);
+        sessionConfig.removeBoard(name);
     }
-    
+
+    /**
+     * Sets the last open board
+     * @param board The name of the last open board.
+     */
     public void setLastOpenBoard(String board) {
-        global.setLastOpenBoard(board);
+        sessionConfig.setLastOpenBoard(board);
     }
-    
+
+    /**
+     * Retrieves the name of the last open board. May be null
```

Comment:
Let's remove `May be null`. The face that we are using `Optional` means it may be `null`.